dist/array_extensions.js

```diff
@@ -0,0 +1,44 @@
+(function (Array) {
+	const EACH_ERROR = 'Function Each already exist';
```

Comment:
I think there is no need for these constants, as more extensions we have the more constants with *almost* the same text we will have.

probably `var extensions` could be some sort of auto generated map from `[each, where, ...]`

```js
var extensions = [each, where].reduce((ext, m) => {
  ext[m.name] = `Function "${m.name}" already exists"`;
  return ext;
}, {});
```